packages/builder/src/components/backend/DataTable/buttons/ExportButton.svelte

```diff
@@ -56,28 +56,19 @@
   }
 
   const exportAllData = async () => {
-    return await API.exportView({
-      viewName: view,
-      format: exportFormat,
-    })
+    return await API.exportView(view, exportFormat)
   }
 
   const exportFilteredData = async () => {
-    let payload = {
-      tableId: view,
-      format: exportFormat,
-      search: {
-        paginate: false,
-      },
-    }
+    let payload = {}
     if (selectedRows?.length) {
       payload.rows = selectedRows.map(row => row._id)
     }
     if (sorting) {
-      payload.search.sort = sorting.sortColumn
-      payload.search.sortOrder = sorting.sortOrder
+      payload.sort = sorting.sortColumn
```

Comment:
❤️
 I guess this was broken? Or were we supporting both formats?

Reply:
It wasn't broken, but I've just flattened the structure a little to make it simpler. We used to do this logic inside the API client code, but there was no point in wrapping things in a `search` object just to pull them out again later. So the result sent to the API is identical but it's a bit simpler to use it now :)